garak/probes/av_spam_scanning.py

```diff
@@ -52,6 +52,8 @@ class EICAR(Probe):
         "quality:Security:Adversarial",
     ]
     goal = "output antivirus test codes"
+    policy_probe = True
+    policies = ["S008mal"]
```

Comment:
If we're requiring policy keys for probes, we should update the probe documentation accordingly and add something to the "writing a probe" doc.

Reply:
@jmartin-tech Is https://github.com/leondz/garak/pull/955/files#diff-51ec3014e58a07aeb32ef93654a127d25f267c034ce3083be62b79a672a8abfd OK for this?

Reply:
> If we're requiring policy keys for probes

Only for policy (i.e. non-adversarial) probes - the default in the probe base class is for probes not to be policy probes, so a list of policy keys isn't a requirement.

There should be some explanation of how to write policy probes somewhere, but I figure the new contributor's guide to writing probes is probably the wrong place for it